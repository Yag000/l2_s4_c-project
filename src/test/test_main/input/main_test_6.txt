mkdir a
mkdir a/b
mkdir a/b/c
mkdir a/b/c/d
cp a cpa
cd a/b/.././b/c/./d/.././.././../../../cpa/b/../b/./c/./d
pwd
cd
cp a/b/.././b/c/./d/.././.././../../../cpa/b/.. a/b/.././b/c/./d/.././../../../copycopya
print
rm a/b/.././b/../b/c/./d/.././../../../copycopya
print
mv a/b/.././b/c/./d/../d/. a/.././../a/b/c/./../c/newd