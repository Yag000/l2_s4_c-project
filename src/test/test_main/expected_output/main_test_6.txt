/cpa/b/c/d
/ (D)
|-- a (D)
|   \-- b (D)
|       \-- c (D)
|           \-- d (D)
|-- cpa (D)
|   \-- b (D)
|       \-- c (D)
|           \-- d (D)
\-- copycopya (D)
    \-- b (D)
        \-- c (D)
            \-- d (D)
/ (D)
|-- a (D)
|   \-- b (D)
|       \-- c (D)
|           \-- d (D)
\-- cpa (D)
    \-- b (D)
        \-- c (D)
            \-- d (D)
